manta-pay/src/simulation/mod.rs

```diff
@@ -136,13 +146,15 @@ impl Simulation {
     where
         L: wallet::test::Ledger<Config> + PublicBalanceOracle<Config>,
         S: wallet::signer::Connection<Config, Checkpoint = L::Checkpoint>,
+        S::Error: Debug,
         GL: FnMut(usize) -> L,
         GS: FnMut(usize) -> S,
         Error<Config, L, S>: Debug,
     {
+        // FIXME: rng
```

Comment:
FIXME

Reply:
Not clear how to fix it, and I don't think it is that important to run the simulation properly. The only issue happens when we want to seed the rng for debugging purposes.